docs/user/introduction.asciidoc

```diff
@@ -70,8 +70,9 @@ image::images/visualization-journey.png[User data analysis journey]
 |===
 
 | *1*
-| *Add data.* The best way to add {es} data to {kib} is to use one of our guided processes,
-available from the <<kibana-home-page,home page>>.
+| *Add data.* The best way to add data to the Elastic Stack is to use one of our many <<connect-to-elasticsearch,integrations>>.
```

Comment:
Can we use `integrations` or is it always `Integrations`?

Reply:
We do use [integrations](https://www.elastic.co/integrations).